scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,229 @@
+
+CONTRIBUTOR_EMAIL = 'contributor@example.com'
+CONTRIBUTOR_USERNAME = 'b'
+
+CLASSROOM_NAME = 'math'
+CLASSROOM_URL_FRAGMENT = 'math'
+
+
+class ContributorDashboardDebugInitializer():
+    """Client-side requests to help develop for the contributor dashboard.
+
+    Attributes:
+        session: object(Session). The requests.Session object to send requests
+            to the Oppia server.
+        base_url: str. The base url of the Oppia server.
+        csrf_token: str. The csrf token of the current session.
+    """
+
+    def __init__(self, base_url: str) -> None:
+        self.session = requests.session()
+        self.base_url = base_url
+        self.csrf_token = ''
+
+    def populate_debug_data(self) -> None:
+        """Populate sample data to help develop for the contributor
+        dashboard.
+        """
+        firebase_admin.initialize_app(
+            options={'projectId': feconf.OPPIA_PROJECT_ID})
+
+        self._sign_up_new_user(SUPER_ADMIN_EMAIL, SUPER_ADMIN_USERNAME)
+        self._sign_up_new_user(CONTRIBUTOR_EMAIL, CONTRIBUTOR_USERNAME)
+
+        self._sign_in(SUPER_ADMIN_EMAIL)
+        self.csrf_token = self._get_csrf_token()
+        self._assign_admin_roles(SUPER_ADMIN_ROLES, SUPER_ADMIN_USERNAME)
+        self._add_submit_question_rights(CONTRIBUTOR_USERNAME)
+        self._generate_sample_new_structures_data()
+        self._add_topics_to_classroom(CLASSROOM_NAME, CLASSROOM_URL_FRAGMENT)
+
+    def _sign_up_new_user(self, email: str, username: str) -> None:
+        """Sign up a new user based on email and username. The password is
+        generated automatically from email.
+        """
+        password = hashlib.md5(email.encode('utf-8')).hexdigest()
+
+        # Create a new user in firebase.
+        firebase_auth.create_user(email=email, password=password) # type: ignore
+
+        # Sign up the new user in Oppia and set its username.
+        self._sign_in(email)
+        self._make_request('GET', '/signup?return_url=/')
+        self.csrf_token = self._get_csrf_token()
+
+        params = {'payload': json.dumps({
+            'username': username,
+            'agreed_to_terms': True,
+            'default_dashboard': constants.DASHBOARD_TYPE_LEARNER,
+        }), 'csrf_token': self.csrf_token}
+
+        self._make_request('POST', feconf.SIGNUP_DATA_URL, params=params)
+
+        # End current session, i.e. log out.
+        self._make_request('GET', '/session_end')
+        self.csrf_token = ''
+
+    def _sign_in(self, email: str) -> None:
+        """Begins a session with the given email, i.e. log in with the email."""
+        password = hashlib.md5(email.encode('utf-8')).hexdigest()
+
+        token_id = self._sign_in_with_email_and_password(email, password)
+        headers = {'Authorization': 'Bearer %s' % token_id}
+
+        self._make_request('GET', '/session_begin', headers=headers)
+
+    def _sign_in_with_email_and_password(
+            self, email: str, password: str) -> Any:
```

Comment:
```suggestion
    def _sign_in_with_email_and_password(
        self, email: str, password: str
    ) -> Any:
```

Please always indent by 4 spaces per the style guide

Reply:
Thanks, I fixed it.